### PR TITLE
Try

### PR DESCRIPTION
**Pull request by LiamRandall**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -104,8 +104,14 @@ URL = "/"
...
 [[Languages.en.menu.main]]
-name = "Get Wasmcloud"
+name = "Try Now"
+URL = "https://www.katacoda.com/wasmcloud#"
 weight = 2
+
+
+[[Languages.en.menu.main]]
+name = "Get Wasmcloud"
+weight = 3
```
Comment (brooksmtownsend) [inline config.toml]: I thought we settled on the following structure, renaming `Get Wasmcloud` to `Get Started`:
```
Get Started
- Try wasmCloud
- Install
- GitHub
```

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]
Only one request, per our standup conversation

File: config.toml (hunk 1 above)
Comment (LiamRandall) [reply config.toml]: I looked at it both ways and liked the stronger call to action here.  Let's see what kind of traction we get with it here; we can move it if not.

Comment (brooksmtownsend) [reply config.toml]: Then perhaps the only change needed is changing capitalization of `Wasmcloud` to `wasmCloud`, unless it should be capitalized in this scenario.

Comment (brooksmtownsend) [reply config.toml]: @LiamRandall ^

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**